app/views/vacancies/_vacancy.html.erb

```diff
@@ -1,7 +1,15 @@
-<li class="vacancy-item">
-  <h3>
+<section class="vacancy-item">
```

Comment:
For me this is incorrect. IMO, `section` should only appear once in a HTML document. What you think @cristofersousa ?